test/compilerlib/CompilerLibTest.cpp

```diff
@@ -57,21 +59,37 @@ void readCommandLineAndKeepUnused(int &argc, char *argv[]) {
   }
 }
 
+// Read the arguments to pass to the testFileName and outputBaseName, other
+// arguments may then be processed by the ONNX-MLIR compiler.
+void readArgsFromCommandLine(int &argc, char *argv[]) {
+  if (std::string(argv[0]) != "onnx-mlir") {
```

Comment:
I don't see why we would test the arg0 here. You can pass `argc` by value, no need for the `&`.